go/client/logclient_test.go

```diff
@@ -214,3 +223,34 @@ func TestAddJSON(t *testing.T) {
 		}
 	}
 }
+
+func TestGetSTHConsistency(t *testing.T) {
+	hs := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		_, err := w.Write([]byte(`{ "consistency": [ "IqlrapPQKtmCY1jCr8+lpCtscRyjjZAA7nyadtFPRFQ=", "ytf6K2GnSRZ3Au+YkivCb7N1DygfKyZmE4aEs9OXl\/8=" ] }`))
```

Comment:
you could inline the test and assignment here since the main return value from w.Write is discarded

if _, err:= w.Write( ...) ; err != nil {
}